### DISCUSSION TITLE
`Find Next` jumps around when not needed, any settings to tweak here?

### DISCUSSION THREAD

**Discussion by metaleap** [orbitalquark/textadept#155] [Q&A] [answered]
Easily observable in a code file where the only handful few find-results of a phrase (say 3-6) all fit within a single screen page. As I keep F3-ing (aka ctrl+G-ing) through them, the buffer view jumps even when all 3 matches are already fully in view. Just somewhat close to the upper/lower boundaries, but not hiding-ly close. Bugs me as I have to carefully discern whether there was "actual" (substantial) movement further-into the file or "just bouncing up&down a little"..

Well I found `view:set_y_caret_policy` and tried all 4 constants with an `y` of 0 (in order to say "I don't need _any_ minimum number-of-lines distance between caret and vertical buffer margins") but what do I know whether this is even interpreted by the (possibly multiple) ways features may move the caret.

Any settings I could set that I have missed so far despite best efforts? Thanks all  =)

**Comment by orbitalquark** [comment, marked as answer]
This happens to me too. I spent some time trying to fix this, but wasn't able :( I've gotten accustomed to looking at the statusbar "Match X/Y" to see if the apparent jump actually moved to another result.

**Reply by metaleap** [reply]
So it's on Scintilla then I guess. Well alright.. can't have everything :grin:
